auto-notify.plugin.zsh

```diff
@@ -37,8 +40,12 @@ function _auto_notify_message() {
 
     if [[ "$platform" == "Linux" ]]; then
         local urgency="normal"
-        if [[ "$exit_code" != "0" ]]; then
-            urgency="critical"
+        if [[ -z $AUTO_NOTIFY_URGENCY ]]; then
+            if [[ "$exit_code" != "0" ]]; then
+                urgency="critical"
+            fi
+        else
+            urgency=$AUTO_NOTIFY_URGENCY
```

Comment:
could you add a test case that shows this works?

Reply:
Unfortunately I don't know zunit and don't have the time to learn it. I personally use the feature so I can guarantee it works, it is only a single if statement after all.

Reply:
It's not too hard if you want to give it a try. If you really don't want to give a go at trying to add a test or two I can do it myself before before merging this to master to the weekend.

Regardless, thanks for the PR :)